checkClock/app/assets/stylesheets/main.scss

```diff
@@ -0,0 +1,6 @@
+@import './scss/clearStyles.scss';
+@import './scss/variables.scss';
+
+@import './scss/home.scss';
+@import './scss/AdminDashboard.scss';
+// @import './scss/login.scss';
```

Comment:
Add a blank line here and if don't need this import commented should be deleted it

Reply:
Done